manifests/c/Crunchyroll/MasterofGarden/1.0.0/Crunchyroll.MasterofGarden.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Crunchyroll.MasterofGarden
+PackageVersion: 1.0.0
+PackageLocale: en-US
+Publisher: Crunchyroll Games
+PackageName: 'The Eminence in Shadow: Master of Garden'

```

```suggestion
PackageName: MasterofGarden
```
This should match control panel

---

Isn’t it supposed to be the full name of the package